app/main/routes.py

```diff
@@ -359,20 +422,39 @@ def search():
         or request.args.get("query", "").lower()
     )
     page = int(request.args.get("page", 1))
+
     filters = {"query": query}
     sorting_orders = build_sorting_orders(request.args)
 
+    breadcrumb_values = {
+        0: {"transferring_body_id": _id},
+        1: {"transferring_body": Body.query.get(_id).Name},
+        2: {"query": ""},
+        3: {"search_terms": ""},
+    }
+
     if query:
+        str_items = ""
```

Comment:
I see we are building the comma separated search terms for the body search results breadcrumbs but neither of the search queries are separating the terms - that needs to be done like we discussed building up lots of OR combinations through a for loop across each term

Reply:
Had call with Terry and it seems like you are doing this as part of the search within search ticket and it will be an AND combination

Reply:
Yes, I am covering it in AYR-765 ticket